for-remote-speakers.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>london.el: The London Emacs Meetup</title>
+        <link href="style.css" rel="stylesheet">
+    </head>
+    <body>
+        <h1>london.el for remote speakers</h1>
+
+	<p>
+          Instructions on what you need to be able to present remotely
+          at the meetup.
+	</p>
+
+	<h2>Connection</h2>
+	<ul>
+	  <li>We can first try Hangouts</li>
+	  <li>Then Skype</li>
+	  <li>Then <a href="https://appear.in/">appear.in</a></li>
+	</ul>
+	<p>
+	  It would be good to have all of the above ready so that if
+	  one approach fails, we can fail over to the next.
+	</p>
+
+	<h2>How long can I speak for?</h2>
+	<p>
+          Tell us how long do you need. Usually we have the venue for
+          90-ish minutes.
```

Comment:
This implies to me that speakers usually talk for ~90 mins. Perhaps say 'with typically 2-3 speakers'.